database/database.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import Column, Integer, PickleType, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+Base = declarative_base()
+
+
+class PersonData(Base):
+    """Declare a mapping for Fellow and Staff objects."""
+
+    __tablename__ = 'person'
+
+    person_id = Column(String, primary_key=True)
+    firstname = Column(String)
```

Comment:
for consistency with other parts of your app (e.g your Person and Dojo class). you might want to use snake case for first_name.last_name and full_name